goodmap/db.py

```
@@ -113,6 +247,16 @@
     return get_locations_list_from_raw_data(self.data, query, location_model)
 
 
+def mongodb_db_get_locations(self, query, location_model):
```

Ruff (0.12.2)

Unused function argument: `location_model`

(ARG001)